mimic/rest/maas_api.py

```diff
@@ -2,7 +2,7 @@
 MAAS Mock API
 """
 
-from __future__ import division, unicode_literals
+from __future__ import division, unicode_literals, print_function
```

Comment:
We definitely don't need it, but this is something I add every time I add a print statement and then tend to leave it behind after I remove the prints; if we *do* end up using a print statement, it makes the code more consistent between 2/3, and since it's a syntactic import it doesn't trigger a py3 warning.